### DISCUSSION TITLE
IOS server notification for Renew and Cancel

### DISCUSSION THREAD

**Discussion by mretcoyy** [imdhemy/laravel-in-app-purchases#195] [Q&A] [answered]
Test Notification for Apple And Google

![image](https://user-images.githubusercontent.com/106091747/200899368-9bbfbbc4-0743-49b1-a369-bab236081367.png)

![image](https://user-images.githubusercontent.com/106091747/200899641-8db17e58-96ab-47fd-be6e-5aa99abecbaf.png)

Did I miss something on setup? why I didnt catch any events
dont mind the missing semicolon. I already updated that. :)

v1.4

**Comment by imdhemy** [comment, marked as answer]
I'm not sure, but just add a `\` before the class name inside the array:

```diff
- App\Listeners\AppleDidRenew::class
+ \App\Listeners\AppleDidRenew::class
```

-- EDIT --

Make sure you are importing the classes the right way, I meant. Bot the event class and the handlers.

**Comment by mretcoyy** [comment]
I already done that but nothing happens. :(

**Comment by mretcoyy** [comment]
I'm sorry Dhemy Its working already thank you!!!

**Reply by imdhemy** [reply]
I'm glad to hear that. Never mind, I'm always happy to help.
